system/temperature.c

```diff
@@ -33,58 +33,81 @@ float cpu_temp_offset = 0;
 // Public Functions
 //------------------------------------------------------------------------------
 
+int TjMax = 0;
+
+uint32_t regl, regh;
```

Comment:
Unless you need to export them later, why not keep them local variables ?

Reply:
1 line saved 😅

Seriously, yes, will move them back.